README.md

```diff
@@ -5,8 +5,8 @@ being sold on eBay. This plugin also exposes the 8 on-board input sensors. You m
 find this plugin on the [Indigo Plugin Store](https://www.indigodomo.com/pluginstore/196/).
 
 This repo includes two technical documents:
-- [Ethnernet IO Controller V5  Instructions.pdf](docs/Ethnernet_IO_Controller_V5_Instructions.pdf)
-- [ZM_TCP Protocol 1.0.pdf](docs/ZM_TCP_Proctocol_1.0.pdf) (misspelled on purpose?)
```

Comment:
I didn't even notice the misspelling in the other line. This is `Proctocol`.

Reply:
I created a pull request to change the filename and link in the readme file.